conf/default-core.yaml

```diff
@@ -3,3 +3,4 @@ core:
   minimize: false
   dist: '{{CWD}}/.dist'
   assetPath: '{{CWD}}'
+  mobileFirst: false
```

Comment:
How would it know which is the default breakpoint? We'd have to sort all the breakpoints by size and compare vs just checking the flag.